test_collections/sdk_tests/support/python_testing/models/rpc_client/test_harness_client.py

```diff
@@ -0,0 +1,71 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import importlib
+import sys
+from multiprocessing.managers import BaseManager
+
+from matter_testing_support import MatterTestConfig, run_tests
+
+try:
+    from matter_yamltests.hooks import TestRunnerHooks
+except:
+
+    class TestRunnerHooks:
+        pass
+
+
+MATTER_DEVELOPMENT_PAA_ROOT_CERTS = "/paa-root-certs"
+
+# Pre-computed param list for each Python Test as defined in Verification Steps.
+test_params = {
+    "TC_ACE_1_3": MatterTestConfig(
+        tests=["test_TC_ACE_1_3"],
```

Comment:
Maybe I'm outdated here, but I remember omitting this `tests=[]` parameter.
Passing the "Test Class" will result running all the tests from the file if `tests` doesn't exist.
And that would keeps this simpler.

Can you try it out @rquidute?

Reply:
Exactly. It should be one test per file.
That's why I'm saying we should omit the `tests=[]` parameter.
Because the test file that should have only one entry point to test.

Reply:
That's not a good assumption - we do have files where there is more than one test_ per file and this is intention because they share a setup

Reply:
The default main scrapes the tests_ by introspection

Reply:
Cecille, can you give more insights about this?

So you're saying we need the `tests=[]` parameter?
If there are python tests with more than one `test_` method, we would have to call more than one?

Reply:
Using the `parse_matter_test_args()` it's great and simpler anyway.
This was changed so closing up.